server/sonar-process-monitor/src/test/java/org/sonar/application/cluster/AppStateClusterImplTest.java

```diff
@@ -116,6 +121,41 @@ public void simulate_network_cluster() throws InterruptedException {
     }
   }
 
+  @Test
+  public void appstateclusterimpl_must_set_sonarqube_version() {
```

Comment:
name should be something like `registerSonarQubeVersion_publishes_version_on_first_call()`